rsts/getting_started.rst

```diff
@@ -80,6 +101,9 @@ Steps
 
   REGISTRY=ghcr.io/flyteorg make fast_register
 
+.. note::
+   If the images are to be re-built, run ``make register`` command.
```

Comment:
register doesn't rebuild the images but just registers the serialized tasks and workflows.
May be you meant reregister here.
Or if you mean rebuilding the cookbook image then that should automatically happen if there are new commits and cookbook:<core | sage_maker|... > sha doesn't exist. Otherwise they can always delete the existing images to rebuild

Reply:
No, `register` does rebuild the cookbook image,`fast_register` doesn't. If you change anything that requires rebuilding the image (to pull the requirements.txt, say), we need to run `make register`.

Reply:
Ok . I think i misunderstood this. Thanks for clarifying.